timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/IntegrationTest.java

```diff
@@ -1575,7 +1577,63 @@ public void createDataWithGraphQl() throws Exception {
         )
       ).toString(), MediaType.APPLICATION_JSON));
 
-    assertThat(queryAfterCreate.readEntity(ObjectNode.class), is(jsnO(
+    ObjectNode queryAfterCreateNode = queryAfterCreate.readEntity(ObjectNode.class);
+    JsonNode hasAdditionListNode = queryAfterCreateNode
+      .get("data")
+      .get("dataSets")
+      .get(dataSetId)
+      .get("schema_Person")
+      .get("tim_pred_latestRevision")
+      .get("_inverse_prov_generated")
+      .get("prov_qualifiedAssociation")
+      .get("prov_hadPlan")
+      .get("tim_pred_additions")
+      .get("tim_pred_hasAdditionList");
+
+    assertThat(hasAdditionListNode.get("items"), containsInAnyOrder(
+      jsnO(
+        "tim_pred_hasKey", jsnO(
+          "uri", jsn("http://schema.org/familyName")
+        ),
+        "tim_pred_hasValueList", jsnO("items", jsnA(
+          jsnO(
+            "tim_pred_rawValue", jsnO(
+              "value", jsn("Test2")
+            ),
+            "tim_pred_type", jsnO(
+              "value", jsn("http://www.w3.org/2001/XMLSchema#string")
+            )
+          ))
+        )
+      ),
+      jsnO(
+        "tim_pred_hasKey", jsnO(
+          "uri", jsn("http://schema.org/givenName")
+        ),
+        "tim_pred_hasValueList", jsnO("items", jsnA(
```

Comment:
Je test hier ook of de volgorde klopt. Ik zou voorstellen om de test te vereenvoudigen en 1 waarde toe tevoegen als "http://schema.org/givenName".

Reply:
Bij deze 1 givenName toegevoegd.